### PR TITLE
Minor fixes and changes to align with documentation.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: InfraDNS/Build.ps1 [hunk 1]
```diff
@@ -44,30 +46,30 @@ Task InstallModules -Depends GenerateEnvironmentFiles {
...
     }
     
-    #End Workaround
+    # End Workaround
     
     foreach ($Resource in $RequiredModules)
     {
-        Install-Module -Name $Resource.Name -RequiredVersion $Resource.Version -Repository 'PSGallery' -Force
-        Save-Module -Name $Resource.Name -RequiredVersion $Resource.Version -Repository 'PSGallery' -Path $ModuleArtifactPath -Force
+        Install-Module -Name $Resource.Name -RequiredVersion $Resource.Version -Force
```
Comment (grayzu) [inline InfraDNS/Build.ps1:64]: Eric, why did you remove the PSGallery from these two lines? I believe that I found that I needed to have it when I ran it in the lab environment. Were you not seeing that behavior?

File: InfraDNS/Configs/DNSServer.ps1 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 configuration DNSServer
 {
-    Import-DscResource -module 'xDnsServer','xNetworking', 'PSDesiredStateConfiguration'
+    Import-DscResource -module 'xDnsServer'
```
Comment (grayzu) [inline InfraDNS/Configs/DNSServer.ps1]: xNetworking is not used so I get why you removed it by why remove PSDesiredStateConfiguration?

File: InfraDNS/Deploy.ps1 [hunk 3]
```diff
@@ -51,14 +51,14 @@ Task DeployModules -Depends Clean {
...
-    #This task uses push to deploy configurations. This task could be used to package up and push configurations to pull server instead.
+    # Start the configuration on TestAgent1. This task could be used to package and place configurations on a pull server instead.
     Start-DscConfiguration -path "$MOFArtifactPath\DevEnv" -Wait -Verbose
 }
 
 Task IntegrationTests -Depends DeployConfigs {
     "Starting Integration tests..."
     #Run Integration tests on target node
-    $Session = New-PSSession -ComputerName TestAgent1
+    $Session = New-PSSession -ComputerName 13.78.178.187
```
Comment (grayzu) [inline InfraDNS/Deploy.ps1]: Can this IP be hard coded in your environment?

**Comment by eslesar** [comment]
I saw the opposite. It wouldn’t run if PSGallery was explicitly named as the Repository, but ran fine if it wasn’t.

File: InfraDNS/Build.ps1 (hunk 1 above)
Comment (ghost) [reply InfraDNS/Build.ps1:64]: I found the opposite to be true. It wouldn't run with PSGallery explicitly set as the Repository, but did fine when it wasn't. I rat-holed on that for a long time.

**Comment by ghost** [comment]
@grayzu Submitted commit to fix the two errors.

**Review by grayzu — COMMENTED** [review]
looks good
